chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -158,9 +158,14 @@ The native window referred to by pname:pCreateInfo->surface must:
 not become associated with a non-Vulkan graphics API surface before
 all associated Vulkan swapchains have been destroyed.
 
-Like core functions, several WSI functions, including
-fname:vkCreateSwapchainKHR return ename:VK_ERROR_DEVICE_LOST if the logical
-device was lost.
+fname:vkCreateSwapchainKHR can return ename:VK_ERROR_DEVICE_LOST if the logical
+device was lost. The sname:VkSwapchainKHR is a child of the device and
+is affected by the lost state; it must: be destroyed before destroying the
+sname:VkDevice. However, sname:VkSurfaceKHR is not a child of any sname:VkDevice
```

Comment:
No need to say vaguely "the device". Could say concretely: pname:device. Same with `pCreateInfo->surface`.

The semicolon seems indulgent punctuation and complicated sentence structure. Maybe: "`pSwapchain` is a child of `device`, and so **must** not be destroyed before `device`.